azure-pipelines.vrt-pr.yml

```diff
@@ -16,21 +16,22 @@ jobs:
   - job: VisualRegressionTest_V7
```

Comment:
## [🕵 FluentUIV7 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28540&commitId=cd693dcc0d6a5fc075aae192be294d784aac4f62&env=prod&clientType=FLUENTUI)


  <details><summary><b>(Dev-Only) Suggestions</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|(Dev-Only) Suggestions.Test of closeButton with overflowing wide custom element.Hovering over the X button on a wide suggestion element.chromium.png|1893| Changed|
|(Dev-Only) Suggestions.Test of closeButton with overflowing wide custom element.default.chromium.png|1919| Changed|
|(Dev-Only) Suggestions.Test of closeButton with overflowing wide custom element.Hovering over a wide suggestion element.chromium.png|1589| Changed|

</details>


  <details><summary><b>ActivityItem</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ActivityItem.Personas.default.chromium.png|54| Changed|
|ActivityItem.Root.default.chromium.png|54| Changed|
|ActivityItem.Root - RTL.default.chromium.png|54| Changed|
|ActivityItem.Personas - RTL.default.chromium.png|54| Changed|

</details>


  <details><summary><b>Breadcrumb</b> 33 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Breadcrumb.Hovering items.actionable overflow hover.chromium.png|210| Changed|
|Breadcrumb.Hovering items.actionable hover.chromium.png|170| Changed|
|Breadcrumb.Button.itemLinkFocus.chromium.png|163| Changed|
|Breadcrumb.Button.click.chromium.png|331| Changed|
|Breadcrumb.Hovering items.non-actionable hover.chromium.png|97| Changed|
|Breadcrumb.Button.shortTitleHover.chromium.png|286| Changed|
|Breadcrumb.Hovering items.non-actionable overflow hover.chromium.png|250| Changed|
|Breadcrumb.Button.hover.chromium.png|166| Changed|
|Breadcrumb.Button.longTitleHover.chromium.png|283| Changed|
|Breadcrumb.Root - RTL.overflowButtonFocus.chromium.png|132| Changed|
|Breadcrumb.Button.default.chromium.png|163| Changed|
|Breadcrumb.Root - RTL.click.chromium.png|273| Changed|
|Breadcrumb.Root - RTL.hover.chromium.png|136| Changed|
|Breadcrumb.Root - RTL.itemLinkFocus.chromium.png|132| Changed|
|Breadcrumb.Root - RTL.default.chromium.png|132| Changed|
|Breadcrumb.Root - RTL.longTitleHover.chromium.png|376| Changed|
|Breadcrumb.Root.hover.chromium.png|136| Changed|
|Breadcrumb.Root.click.chromium.png|273| Changed|
|Breadcrumb.Root - RTL.shortTitleHover.chromium.png|232| Changed|
|Breadcrumb.Button.overflowButtonFocus.chromium.png|163| Changed|
|Breadcrumb.Root.longTitleHover.chromium.png|376| Changed|
|Breadcrumb.Root.default.chromium.png|132| Changed|
|Breadcrumb.Root.overflowButtonFocus.chromium.png|132| Changed|
|Breadcrumb.Root.itemLinkFocus.chromium.png|132| Changed|
|Breadcrumb.Root.shortTitleHover.chromium.png|232| Changed|
|Breadcrumb.Hovering items.chromium.png|97| Changed|
|Breadcrumb.Button - RTL.click.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.default.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.hover.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.itemLinkFocus.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.longTitleHover.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.overflowButtonFocus.chromium.png|0| Removed|
|Breadcrumb.Button - RTL.shortTitleHover.chromium.png|0| Removed|

</details>


  <details><summary><b>Button Action</b> 13 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Action.Disabled.default.chromium.png|29| Changed|
|Button Action.Disabled.hover.chromium.png|29| Changed|
|Button Action.Checked.default.chromium.png|7| Changed|
|Button Action.Disabled.pressed.chromium.png|29| Changed|
|Button Action.Icon Only.default.chromium.png|5| Changed|
|Button Action.Icon Only.pressed.chromium.png|7| Changed|
|Button Action.Checked.hover.chromium.png|7| Changed|
|Button Action.Root - RTL.default.chromium.png|44| Changed|
|Button Action.Root.pressed.chromium.png|7| Changed|
|Button Action.Checked.pressed.chromium.png|7| Changed|
|Button Action.No Icon.default.chromium.png|39| Changed|
|Button Action.Root.default.chromium.png|44| Changed|
|Button Action.Root - RTL.pressed.chromium.png|7| Changed|

</details>


  <details><summary><b>Button Command</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Command.Checked.hover.chromium.png|587| Changed|
|Button Command.Disabled.hover.chromium.png|43| Changed|
|Button Command.Checked.pressed.chromium.png|587| Changed|
|Button Command.Disabled.default.chromium.png|43| Changed|
|Button Command.Disabled.pressed.chromium.png|43| Changed|
|Button Command.Root.open.chromium.png|747| Changed|
|Button Command.Root - RTL.open.chromium.png|747| Changed|
|Button Command.Disabled.open.chromium.png|43| Changed|
|Button Command.Root - RTL.pressed.chromium.png|4| Changed|
|Button Command.Root - RTL.default.chromium.png|76| Changed|
|Button Command.Root.pressed.chromium.png|4| Changed|
|Button Command.Root.default.chromium.png|76| Changed|
|Button Command.Root.hover.chromium.png|8| Changed|
|Button Command.Checked.default.chromium.png|4| Changed|
|Button Command.Checked.open.chromium.png|747| Changed|
|Button Command.Root - RTL.hover.chromium.png|8| Changed|

</details>


  <details><summary><b>Button Compound</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Compound.No Icon.default.chromium.png|53| Changed|
|Button Compound.Primary.hover.chromium.png|2611| Changed|
|Button Compound.Primary Disabled.hover.chromium.png|24| Changed|
|Button Compound.Primary Disabled.pressed.chromium.png|24| Changed|
|Button Compound.Root - RTL.default.chromium.png|97| Changed|
|Button Compound.Disabled.hover.chromium.png|221| Changed|
|Button Compound.Disabled.default.chromium.png|221| Changed|
|Button Compound.Disabled.pressed.chromium.png|221| Changed|
|Button Compound.Root.default.chromium.png|97| Changed|
|Button Compound.Primary Disabled.default.chromium.png|24| Changed|

</details>


  <details><summary><b>Button Default</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Default.Disabled.default.chromium.png|37| Changed|
|Button Default.No Icon.default.chromium.png|53| Changed|
|Button Default.Icon Only.default.chromium.png|14| Changed|
|Button Default.Disabled.pressed.chromium.png|37| Changed|
|Button Default.Primary Disabled.pressed.chromium.png|8| Changed|
|Button Default.Primary Disabled.hover.chromium.png|8| Changed|
|Button Default.Disabled.hover.chromium.png|37| Changed|
|Button Default.Primary Anchor.hover.chromium.png|324| Changed|
|Button Default.Primary Disabled.default.chromium.png|8| Changed|
|Button Default.Root - RTL.default.chromium.png|67| Changed|
|Button Default.Root.default.chromium.png|67| Changed|
|Button Default.Primary.hover.chromium.png|417| Changed|

</details>


  <details><summary><b>Button Next</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Next.Anchor.focus.chromium.png|85| Changed|
|Button Next.Default.default.chromium.png|85| Changed|
|Button Next.Disabled.focus.chromium.png|47| Changed|
|Button Next.Disabled.hover.chromium.png|47| Changed|
|Button Next.Ghost Disabled.default.chromium.png|34| Changed|
|Button Next.Ghost Disabled.hover.chromium.png|34| Changed|
|Button Next.Ghost Disabled.pressed.chromium.png|34| Changed|
|Button Next.Disabled.pressed.chromium.png|47| Changed|
|Button Next.Ghost.default.chromium.png|62| Changed|
|Button Next.Primary Disabled.default.chromium.png|47| Changed|
|Button Next.Primary Disabled.pressed.chromium.png|47| Changed|
|Button Next.Primary Disabled.focus.chromium.png|47| Changed|
|Button Next.Transparent Disabled.default.chromium.png|34| Changed|
|Button Next.Ghost.focus.chromium.png|62| Changed|
|Button Next.Transparent.default.chromium.png|62| Changed|
|Button Next.Default.focus.chromium.png|85| Changed|
|Button Next.Transparent Disabled.hover.chromium.png|34| Changed|
|Button Next.Transparent Disabled.focus.chromium.png|34| Changed|
|Button Next.Disabled.default.chromium.png|47| Changed|
|Button Next.Ghost Disabled.focus.chromium.png|34| Changed|
|Button Next.Primary Disabled.hover.chromium.png|47| Changed|
|Button Next.Transparent Disabled.pressed.chromium.png|34| Changed|
|Button Next.Transparent.focus.chromium.png|62| Changed|
|Button Next.Anchor.default.chromium.png|85| Changed|

</details>


  <details><summary><b>Button Next - Block</b> 25 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Next - Block.Default.default.chromium.png|95| Changed|
|Button Next - Block.Ghost.focus.chromium.png|66| Changed|
|Button Next - Block.Default.focus.chromium.png|95| Changed|
|Button Next - Block.Ghost Disabled.default.chromium.png|40| Changed|
|Button Next - Block.Disabled.pressed.chromium.png|53| Changed|
|Button Next - Block.Ghost Disabled.focus.chromium.png|40| Changed|
|Button Next - Block.Disabled.default.chromium.png|53| Changed|
|Button Next - Block.Ghost Disabled.hover.chromium.png|40| Changed|
|Button Next - Block.Primary Disabled.focus.chromium.png|53| Changed|
|Button Next - Block.Disabled.focus.chromium.png|53| Changed|
|Button Next - Block.Ghost.default.chromium.png|66| Changed|
|Button Next - Block.Primary Disabled.hover.chromium.png|53| Changed|
|Button Next - Block.Primary Disabled.default.chromium.png|53| Changed|
|Button Next - Block.Primary.hover.chromium.png|7706| Changed|
|Button Next - Block.Primary Disabled.pressed.chromium.png|53| Changed|
|Button Next - Block.Ghost Disabled.pressed.chromium.png|40| Changed|
|Button Next - Block.Ghost.pressed.chromium.png|5| Changed|
|Button Next - Block.Disabled.hover.chromium.png|53| Changed|
|Button Next - Block.Transparent Disabled.focus.chromium.png|40| Changed|
|Button Next - Block.Transparent Disabled.pressed.chromium.png|40| Changed|
|Button Next - Block.Transparent Disabled.default.chromium.png|40| Changed|
|Button Next - Block.Transparent.default.chromium.png|66| Changed|
|Button Next - Block.Transparent.pressed.chromium.png|5| Changed|
|Button Next - Block.Transparent.focus.chromium.png|66| Changed|
|Button Next - Block.Transparent Disabled.hover.chromium.png|40| Changed|

</details>


  <details><summary><b>Button Next - Circular</b> 25 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Next - Circular.Disabled.default.chromium.png|53| Changed|
|Button Next - Circular.Disabled.focus.chromium.png|53| Changed|
|Button Next - Circular.Disabled.pressed.chromium.png|53| Changed|
|Button Next - Circular.Ghost Disabled.default.chromium.png|40| Changed|
|Button Next - Circular.Ghost Disabled.focus.chromium.png|40| Changed|
|Button Next - Circular.Ghost.default.chromium.png|66| Changed|
|Button Next - Circular.Ghost Disabled.hover.chromium.png|40| Changed|
|Button Next - Circular.Ghost.pressed.chromium.png|5| Changed|
|Button Next - Circular.Primary Disabled.focus.chromium.png|53| Changed|
|Button Next - Circular.Primary Disabled.hover.chromium.png|53| Changed|
|Button Next - Circular.Ghost Disabled.pressed.chromium.png|40| Changed|
|Button Next - Circular.Disabled.hover.chromium.png|53| Changed|
|Button Next - Circular.Primary Disabled.pressed.chromium.png|53| Changed|
|Button Next - Circular.Default.default.chromium.png|95| Changed|
|Button Next - Circular.Primary.hover.chromium.png|518| Changed|
|Button Next - Circular.Transparent Disabled.focus.chromium.png|40| Changed|
|Button Next - Circular.Ghost.focus.chromium.png|66| Changed|
|Button Next - Circular.Transparent Disabled.hover.chromium.png|40| Changed|
|Button Next - Circular.Transparent.focus.chromium.png|66| Changed|
|Button Next - Circular.Transparent.pressed.chromium.png|5| Changed|
|Button Next - Circular.Primary Disabled.default.chromium.png|53| Changed|
|Button Next - Circular.Transparent Disabled.default.chromium.png|40| Changed|
|Button Next - Circular.Transparent Disabled.pressed.chromium.png|40| Changed|
|Button Next - Circular.Default.focus.chromium.png|95| Changed|
|Button Next - Circular.Transparent.default.chromium.png|66| Changed|

</details>


  <details><summary><b>Button Next - Icon only</b> 25 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Next - Icon only.Default.default.chromium.png|11| Changed|
|Button Next - Icon only.Disabled.focus.chromium.png|6| Changed|
|Button Next - Icon only.Disabled.hover.chromium.png|6| Changed|
|Button Next - Icon only.Ghost Disabled.default.chromium.png|6| Changed|
|Button Next - Icon only.Ghost Disabled.pressed.chromium.png|6| Changed|
|Button Next - Icon only.Disabled.pressed.chromium.png|6| Changed|
|Button Next - Icon only.Disabled.default.chromium.png|6| Changed|
|Button Next - Icon only.Default.focus.chromium.png|11| Changed|
|Button Next - Icon only.Ghost.pressed.chromium.png|5| Changed|
|Button Next - Icon only.Ghost Disabled.focus.chromium.png|6| Changed|
|Button Next - Icon only.Ghost Disabled.hover.chromium.png|6| Changed|
|Button Next - Icon only.Primary Disabled.default.chromium.png|6| Changed|
|Button Next - Icon only.Transparent Disabled.default.chromium.png|6| Changed|
|Button Next - Icon only.Primary.hover.chromium.png|127| Changed|
|Button Next - Icon only.Transparent Disabled.hover.chromium.png|6| Changed|
|Button Next - Icon only.Ghost.focus.chromium.png|4| Changed|
|Button Next - Icon only.Transparent Disabled.pressed.chromium.png|6| Changed|
|Button Next - Icon only.Transparent.pressed.chromium.png|5| Changed|
|Button Next - Icon only.Ghost.default.chromium.png|4| Changed|
|Button Next - Icon only.Transparent Disabled.focus.chromium.png|6| Changed|
|Button Next - Icon only.Transparent.focus.chromium.png|4| Changed|
|Button Next - Icon only.Primary Disabled.pressed.chromium.png|6| Changed|
|Button Next - Icon only.Primary Disabled.focus.chromium.png|6| Changed|
|Button Next - Icon only.Primary Disabled.hover.chromium.png|6| Changed|
|Button Next - Icon only.Transparent.default.chromium.png|4| Changed|

</details>


  <details><summary><b>Button Next - Inverted</b> 25 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Next - Inverted.Disabled.default.chromium.png|53| Changed|
|Button Next - Inverted.Default.default.chromium.png|95| Changed|
|Button Next - Inverted.Ghost Disabled.hover.chromium.png|40| Changed|
|Button Next - Inverted.Ghost Disabled.pressed.chromium.png|40| Changed|
|Button Next - Inverted.Ghost.focus.chromium.png|66| Changed|
|Button Next - Inverted.Primary Disabled.hover.chromium.png|53| Changed|
|Button Next - Inverted.Primary Disabled.pressed.chromium.png|53| Changed|
|Button Next - Inverted.Disabled.focus.chromium.png|53| Changed|
|Button Next - Inverted.Primary Disabled.default.chromium.png|53| Changed|
|Button Next - Inverted.Primary.hover.chromium.png|546| Changed|
|Button Next - Inverted.Transparent Disabled.focus.chromium.png|40| Changed|
|Button Next - Inverted.Primary Disabled.focus.chromium.png|53| Changed|
|Button Next - Inverted.Ghost Disabled.default.chromium.png|40| Changed|
|Button Next - Inverted.Transparent Disabled.pressed.chromium.png|40| Changed|
|Button Next - Inverted.Transparent.focus.chromium.png|66| Changed|
|Button Next - Inverted.Transparent Disabled.default.chromium.png|40| Changed|
|Button Next - Inverted.Disabled.pressed.chromium.png|53| Changed|
|Button Next - Inverted.Disabled.hover.chromium.png|53| Changed|
|Button Next - Inverted.Transparent.pressed.chromium.png|5| Changed|
|Button Next - Inverted.Transparent Disabled.hover.chromium.png|40| Changed|
|Button Next - Inverted.Transparent.default.chromium.png|66| Changed|
|Button Next - Inverted.Ghost Disabled.focus.chromium.png|40| Changed|
|Button Next - Inverted.Ghost.pressed.chromium.png|5| Changed|
|Button Next - Inverted.Default.focus.chromium.png|95| Changed|
|Button Next - Inverted.Ghost.default.chromium.png|66| Changed|

</details>


  <details><summary><b>Button Next - Loading</b> 32 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Next - Loading.Default.hover.chromium.png|53| Changed|
|Button Next - Loading.Default.focus.chromium.png|53| Changed|
|Button Next - Loading.Disabled.default.chromium.png|53| Changed|
|Button Next - Loading.Disabled.hover.chromium.png|53| Changed|
|Button Next - Loading.Disabled.focus.chromium.png|53| Changed|
|Button Next - Loading.Disabled.pressed.chromium.png|53| Changed|
|Button Next - Loading.Ghost Disabled.focus.chromium.png|40| Changed|
|Button Next - Loading.Ghost Disabled.hover.chromium.png|40| Changed|
|Button Next - Loading.Default.pressed.chromium.png|53| Changed|
|Button Next - Loading.Ghost Disabled.pressed.chromium.png|40| Changed|
|Button Next - Loading.Ghost.focus.chromium.png|40| Changed|
|Button Next - Loading.Ghost Disabled.default.chromium.png|40| Changed|
|Button Next - Loading.Ghost.default.chromium.png|40| Changed|
|Button Next - Loading.Ghost.hover.chromium.png|40| Changed|
|Button Next - Loading.Primary Disabled.default.chromium.png|53| Changed|
|Button Next - Loading.Primary Disabled.focus.chromium.png|53| Changed|
|Button Next - Loading.Primary Disabled.hover.chromium.png|53| Changed|
|Button Next - Loading.Ghost.pressed.chromium.png|40| Changed|
|Button Next - Loading.Primary.default.chromium.png|53| Changed|
|Button Next - Loading.Primary.focus.chromium.png|53| Changed|
|Button Next - Loading.Transparent Disabled.default.chromium.png|40| Changed|
|Button Next - Loading.Transparent Disabled.hover.chromium.png|40| Changed|
|Button Next - Loading.Primary.pressed.chromium.png|53| Changed|
|Button Next - Loading.Transparent Disabled.focus.chromium.png|40| Changed|
|Button Next - Loading.Transparent.default.chromium.png|40| Changed|
|Button Next - Loading.Transparent Disabled.pressed.chromium.png|40| Changed|
|Button Next - Loading.Transparent.focus.chromium.png|40| Changed|
|Button Next - Loading.Transparent.pressed.chromium.png|40| Changed|
|Button Next - Loading.Transparent.hover.chromium.png|40| Changed|
|Button Next - Loading.Default.default.chromium.png|53| Changed|
|Button Next - Loading.Primary.hover.chromium.png|53| Changed|
|Button Next - Loading.Primary Disabled.pressed.chromium.png|53| Changed|

</details>


  <details><summary><b>Button Next - Sizes</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Next - Sizes.Larger.default.chromium.png|95| Changed|
|Button Next - Sizes.Largest.default.chromium.png|95| Changed|
|Button Next - Sizes.Largest.focus.chromium.png|95| Changed|
|Button Next - Sizes.Small.default.chromium.png|95| Changed|
|Button Next - Sizes.Smaller.default.chromium.png|95| Changed|
|Button Next - Sizes.Smaller.focus.chromium.png|95| Changed|
|Button Next - Sizes.Large.focus.chromium.png|95| Changed|
|Button Next - Sizes.Smallest.default.chromium.png|95| Changed|
|Button Next - Sizes.Small.focus.chromium.png|95| Changed|
|Button Next - Sizes.Larger.focus.chromium.png|95| Changed|
|Button Next - Sizes.Large.default.chromium.png|95| Changed|
|Button Next - Sizes.Smallest.focus.chromium.png|95| Changed|

</details>


  <details><summary><b>Button Next - With icon after content</b> 25 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Next - With icon after content.Default.focus.chromium.png|95| Changed|
|Button Next - With icon after content.Disabled.default.chromium.png|53| Changed|
|Button Next - With icon after content.Disabled.pressed.chromium.png|53| Changed|
|Button Next - With icon after content.Disabled.focus.chromium.png|53| Changed|
|Button Next - With icon after content.Ghost Disabled.default.chromium.png|40| Changed|
|Button Next - With icon after content.Ghost Disabled.focus.chromium.png|40| Changed|
|Button Next - With icon after content.Ghost Disabled.hover.chromium.png|40| Changed|
|Button Next - With icon after content.Primary Disabled.default.chromium.png|53| Changed|
|Button Next - With icon after content.Primary Disabled.focus.chromium.png|53| Changed|
|Button Next - With icon after content.Primary Disabled.hover.chromium.png|53| Changed|
|Button Next - With icon after content.Primary.hover.chromium.png|546| Changed|
|Button Next - With icon after content.Ghost.focus.chromium.png|66| Changed|
|Button Next - With icon after content.Transparent Disabled.focus.chromium.png|40| Changed|
|Button Next - With icon after content.Transparent Disabled.pressed.chromium.png|40| Changed|
|Button Next - With icon after content.Transparent.default.chromium.png|66| Changed|
|Button Next - With icon after content.Transparent.focus.chromium.png|66| Changed|
|Button Next - With icon after content.Transparent.pressed.chromium.png|5| Changed|
|Button Next - With icon after content.Default.default.chromium.png|95| Changed|
|Button Next - With icon after content.Transparent Disabled.hover.chromium.png|40| Changed|
|Button Next - With icon after content.Primary Disabled.pressed.chromium.png|53| Changed|
|Button Next - With icon after content.Ghost.default.chromium.png|66| Changed|
|Button Next - With icon after content.Ghost.pressed.chromium.png|5| Changed|
|Button Next - With icon after content.Transparent Disabled.default.chromium.png|40| Changed|
|Button Next - With icon after content.Ghost Disabled.pressed.chromium.png|40| Changed|
|Button Next - With icon after content.Disabled.hover.chromium.png|53| Changed|

</details>


  <details><summary><b>Button Next - With icon before content</b> 25 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Next - With icon before content.Default.default.chromium.png|95| Changed|
|Button Next - With icon before content.Default.focus.chromium.png|95| Changed|
|Button Next - With icon before content.Disabled.focus.chromium.png|53| Changed|
|Button Next - With icon before content.Disabled.pressed.chromium.png|53| Changed|
|Button Next - With icon before content.Ghost Disabled.focus.chromium.png|40| Changed|
|Button Next - With icon before content.Ghost Disabled.default.chromium.png|40| Changed|
|Button Next - With icon before content.Disabled.default.chromium.png|53| Changed|
|Button Next - With icon before content.Ghost.default.chromium.png|66| Changed|
|Button Next - With icon before content.Ghost.focus.chromium.png|66| Changed|
|Button Next - With icon before content.Ghost.pressed.chromium.png|5| Changed|
|Button Next - With icon before content.Primary Disabled.default.chromium.png|53| Changed|
|Button Next - With icon before content.Ghost Disabled.pressed.chromium.png|40| Changed|
|Button Next - With icon before content.Primary Disabled.hover.chromium.png|53| Changed|
|Button Next - With icon before content.Ghost Disabled.hover.chromium.png|40| Changed|
|Button Next - With icon before content.Primary Disabled.pressed.chromium.png|53| Changed|
|Button Next - With icon before content.Transparent Disabled.default.chromium.png|40| Changed|
|Button Next - With icon before content.Primary Disabled.focus.chromium.png|53| Changed|
|Button Next - With icon before content.Primary.hover.chromium.png|546| Changed|
|Button Next - With icon before content.Transparent Disabled.focus.chromium.png|40| Changed|
|Button Next - With icon before content.Disabled.hover.chromium.png|53| Changed|
|Button Next - With icon before content.Transparent Disabled.pressed.chromium.png|40| Changed|
|Button Next - With icon before content.Transparent.focus.chromium.png|66| Changed|
|Button Next - With icon before content.Transparent Disabled.hover.chromium.png|40| Changed|
|Button Next - With icon before content.Transparent.default.chromium.png|66| Changed|
|Button Next - With icon before content.Transparent.pressed.chromium.png|5| Changed|

</details>


  <details><summary><b>Button Next - With styled icon from react-icons via tokens</b> 25 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Next - With styled icon from react-icons via tokens.Default.default.chromium.png|40| Changed|
|Button Next - With styled icon from react-icons via tokens.Default.focus.chromium.png|37| Changed|
|Button Next - With styled icon from react-icons via tokens.Disabled.default.chromium.png|22| Changed|
|Button Next - With styled icon from react-icons via tokens.Disabled.pressed.chromium.png|22| Changed|
|Button Next - With styled icon from react-icons via tokens.Ghost.focus.chromium.png|14| Changed|
|Button Next - With styled icon from react-icons via tokens.Ghost.pressed.chromium.png|18| Changed|
|Button Next - With styled icon from react-icons via tokens.Primary Disabled.focus.chromium.png|22| Changed|
|Button Next - With styled icon from react-icons via tokens.Primary Disabled.pressed.chromium.png|22| Changed|
|Button Next - With styled icon from react-icons via tokens.Ghost Disabled.default.chromium.png|22| Changed|
|Button Next - With styled icon from react-icons via tokens.Primary.hover.chromium.png|379| Changed|
|Button Next - With styled icon from react-icons via tokens.Transparent Disabled.default.chromium.png|22| Changed|
|Button Next - With styled icon from react-icons via tokens.Transparent Disabled.focus.chromium.png|22| Changed|
|Button Next - With styled icon from react-icons via tokens.Transparent Disabled.hover.chromium.png|22| Changed|
|Button Next - With styled icon from react-icons via tokens.Transparent Disabled.pressed.chromium.png|22| Changed|
|Button Next - With styled icon from react-icons via tokens.Transparent.default.chromium.png|15| Changed|
|Button Next - With styled icon from react-icons via tokens.Transparent.pressed.chromium.png|18| Changed|
|Button Next - With styled icon from react-icons via tokens.Transparent.focus.chromium.png|14| Changed|
|Button Next - With styled icon from react-icons via tokens.Disabled.hover.chromium.png|22| Changed|
|Button Next - With styled icon from react-icons via tokens.Ghost Disabled.hover.chromium.png|22| Changed|
|Button Next - With styled icon from react-icons via tokens.Primary Disabled.default.chromium.png|22| Changed|
|Button Next - With styled icon from react-icons via tokens.Primary Disabled.hover.chromium.png|22| Changed|
|Button Next - With styled icon from react-icons via tokens.Disabled.focus.chromium.png|22| Changed|
|Button Next - With styled icon from react-icons via tokens.Ghost.default.chromium.png|15| Changed|
|Button Next - With styled icon from react-icons via tokens.Ghost Disabled.focus.chromium.png|22| Changed|
|Button Next - With styled icon from react-icons via tokens.Ghost Disabled.pressed.chromium.png|22| Changed|

</details>


  <details><summary><b>Button Special Scenarios</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Special Scenarios.no flex shrink.default.chromium.png|451| Changed|

</details>


  <details><summary><b>Button Split</b> 26 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Split.Command Split.default.chromium.png|80| Changed|
|Button Split.Command Split.hover split.chromium.png|80| Changed|
|Button Split.Checked.hover main.chromium.png|94| Changed|
|Button Split.Default with Primary Action Disabled.default.chromium.png|65| Changed|
|Button Split.Disabled.hover split.chromium.png|63| Changed|
|Button Split.Default with Primary Action Disabled.hover split.chromium.png|65| Changed|
|Button Split.Disabled.default.chromium.png|63| Changed|
|Button Split.Checked.hover split.chromium.png|94| Changed|
|Button Split.Checked.default.chromium.png|94| Changed|
|Button Split.Primary.hover main.chromium.png|605| Changed|
|Button Split.Primary with Primary Action Disabled.hover main.chromium.png|61| Changed|
|Button Split.Command Split.hover main.chromium.png|8| Changed|
|Button Split.Primary.hover split.chromium.png|126| Changed|
|Button Split.Root - RTL.hover split.chromium.png|114| Changed|
|Button Split.Root.hover main.chromium.png|4| Changed|
|Button Split.Root - RTL.default.chromium.png|114| Changed|
|Button Split.Primary with Primary Action Disabled.hover split.chromium.png|186| Changed|
|Button Split.Primary Disabled.hover main.chromium.png|8| Changed|
|Button Split.Root.default.chromium.png|114| Changed|
|Button Split.Root - RTL.hover main.chromium.png|4| Changed|
|Button Split.Root.hover split.chromium.png|114| Changed|
|Button Split.Disabled.hover main.chromium.png|63| Changed|
|Button Split.Primary Disabled.default.chromium.png|8| Changed|
|Button Split.Primary Disabled.hover split.chromium.png|8| Changed|
|Button Split.Default with Primary Action Disabled.hover main.chromium.png|65| Changed|
|Button Split.Primary with Primary Action Disabled.default.chromium.png|6| Changed|

</details>


  <details><summary><b>Calendar</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Calendar.Root.default.chromium.png|27700| Changed|
|Calendar.Root - RTL.default.chromium.png|27704| Changed|

</details>


  <details><summary><b>Calendar - No Month Option</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Calendar - No Month Option.Show Month as Overlay and no Go To Today.default.chromium.png|27304| Changed|

</details>


  <details><summary><b>Callout</b> 22 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Beak 25.default.chromium.png|569| Changed|
|Callout.Left top edge.default.chromium.png|569| Changed|
|Callout.Bottom right edge - RTL.default.chromium.png|569| Changed|
|Callout.Gap space 25.default.chromium.png|569| Changed|
|Callout.Bottom center.default.chromium.png|569| Changed|
|Callout.Bottom auto edge.default.chromium.png|569| Changed|
|Callout.Left bottom edge.default.chromium.png|569| Changed|
|Callout.No beak.default.chromium.png|569| Changed|
|Callout.Bottom right edge.default.chromium.png|569| Changed|
|Callout.Bottom left edge.default.chromium.png|569| Changed|
|Callout.Right bottom edge.default.chromium.png|569| Changed|
|Callout.Bottom left edge - RTL.default.chromium.png|569| Changed|
|Callout.No callout width specified.default.chromium.png|569| Changed|
|Callout.Rendering callout attached to a rectangle.default.chromium.png|569| Changed|
|Callout.Left center.default.chromium.png|569| Changed|
|Callout.Right center.default.chromium.png|569| Changed|
|Callout.Top auto edge.default.chromium.png|569| Changed|
|Callout.Root.default.chromium.png|569| Changed|
|Callout.Top left edge.default.chromium.png|569| Changed|
|Callout.Top right edge.default.chromium.png|569| Changed|
|Callout.Right top edge.default.chromium.png|569| Changed|
|Callout.Top center.default.chromium.png|569| Changed|

</details>


  <details><summary><b>Card</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card.Horizontal Card - Basic - Hoverable.focus.chromium.png|108| Changed|
|Card.Horizontal Card - Basic - Hoverable.default.chromium.png|108| Changed|
|Card.Horizontal Card - Basic - Hoverable.hover.chromium.png|108| Changed|
|Card.Horizontal Card - Basic - Non hoverable.hover.chromium.png|108| Changed|
|Card.Horizontal Card - Basic - Non hoverable.focus.chromium.png|108| Changed|
|Card.Vertical Card - Basic - Hoverable.focus.chromium.png|92| Changed|
|Card.Horizontal Card - Basic - Non hoverable.default.chromium.png|108| Changed|
|Card.Vertical Card - Basic - Non hoverable.hover.chromium.png|92| Changed|
|Card.Vertical Card - Basic - Non hoverable.focus.chromium.png|92| Changed|
|Card.Vertical Card - Basic - Hoverable.default.chromium.png|92| Changed|
|Card.Vertical Card - Basic - Non hoverable.default.chromium.png|92| Changed|
|Card.Vertical Card - Basic - Hoverable.hover.chromium.png|92| Changed|
|Card.Vertical Card - Example with contents - Image in middle.default.chromium.png|65| Changed|
|Card.Vertical Card - Example with contents - Image in middle.focus.chromium.png|65| Changed|
|Card.Vertical Card - Example with contents - Image in middle.hover.chromium.png|65| Changed|
|Card.Vertical Card - Example with contents - Image on top.default.chromium.png|45220| Changed|
|Card.Vertical Card - Example with contents - Image on top.hover.chromium.png|45204| Changed|
|Card.Vertical Card - Example with contents - Image on top.focus.chromium.png|45174| Changed|

</details>


  <details><summary><b>Checkbox</b> 13 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Checked disabled.default.chromium.png|82| Changed|
|Checkbox.Checked.default.chromium.png|100| Changed|
|Checkbox.Controlled Indeterminate disabled.default.chromium.png|129| Changed|
|Checkbox.Controlled Indeterminate.default.chromium.png|187| Changed|
|Checkbox.End - RTL.default.chromium.png|100| Changed|
|Checkbox.End.default.chromium.png|100| Changed|
|Checkbox.Custom render Checkbox.default.chromium.png|11697| Changed|
|Checkbox.Unchecked - RTL.default.chromium.png|137| Changed|
|Checkbox.Unchecked disabled.default.chromium.png|89| Changed|
|Checkbox.Unchecked.default.chromium.png|137| Changed|
|Checkbox.Uncontrolled Indeterminate.default.chromium.png|200| Changed|
|Checkbox.Uncontrolled Indeterminate disabled.default.chromium.png|136| Changed|
|Checkbox.Multi-line Checkbox.default.chromium.png|6128| Changed|

</details>


  <details><summary><b>Checkbox Indeterminate</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox Indeterminate.Controlled Indeterminate checkbox without onChange callback.default.chromium.png|332| Changed|
|Checkbox Indeterminate.Uncontrolled Indeterminate checkbox.default.chromium.png|200| Changed|
|Checkbox Indeterminate.Uncontrolled Indeterminate checkbox.clicked.chromium.png|25| Changed|
|Checkbox Indeterminate.Uncontrolled Indeterminate defaultChecked checkbox.default.chromium.png|290| Changed|
|Checkbox Indeterminate.Uncontrolled and Controlled Indeterminate checkbox.default.chromium.png|278| Changed|

</details>


  <details><summary><b>ChoiceGroup</b> 24 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ChoiceGroup.Required.default.chromium.png|232| Changed|
|ChoiceGroup.Required.hover unselected.chromium.png|183| Changed|
|ChoiceGroup.Root.default.chromium.png|232| Changed|
|ChoiceGroup.Required.selected.chromium.png|166| Changed|
|ChoiceGroup.Root.hover selected.chromium.png|166| Changed|
|ChoiceGroup.Root.selected.chromium.png|166| Changed|
|ChoiceGroup.Root.hover unselected.chromium.png|183| Changed|
|ChoiceGroup.With default size images.hover unselected.chromium.png|294| Changed|
|ChoiceGroup.With default size images.hover selected.chromium.png|150| Changed|
|ChoiceGroup.With default size images.selected.chromium.png|150| Changed|
|ChoiceGroup.Required.hover selected.chromium.png|166| Changed|
|ChoiceGroup.With icons - RTL.hover unselected.chromium.png|605| Changed|
|ChoiceGroup.With default size images.default.chromium.png|249| Changed|
|ChoiceGroup.With icons - RTL.hover selected.chromium.png|461| Changed|
|ChoiceGroup.With icons - RTL.default.chromium.png|485| Changed|
|ChoiceGroup.With icons.selected.chromium.png|461| Changed|
|ChoiceGroup.With icons.hover unselected.chromium.png|605| Changed|
|ChoiceGroup.With icons - RTL.selected.chromium.png|461| Changed|
|ChoiceGroup.With icons.default.chromium.png|485| Changed|
|ChoiceGroup.With icons.hover selected.chromium.png|461| Changed|
|ChoiceGroup.With large size images.hover unselected.chromium.png|406| Changed|
|ChoiceGroup.With large size images.default.chromium.png|249| Changed|
|ChoiceGroup.With large size images.hover selected.chromium.png|150| Changed|
|ChoiceGroup.With large size images.selected.chromium.png|150| Changed|

</details>


  <details><summary><b>Coachmark</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Coachmark.Collapsed.default.chromium.png|62| Changed|

</details>


  <details><summary><b>ColorPicker</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ColorPicker.Blue.default.chromium.png|10396| Changed|
|ColorPicker.Root - RTL.default.chromium.png|10369| Changed|
|ColorPicker.Root.default.chromium.png|10368| Changed|

</details>


  <details><summary><b>ComboBox</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ComboBox.Disabled.default.chromium.png|74| Changed|
|ComboBox.Error.default.chromium.png|92| Changed|
|ComboBox.Placeholder.default.chromium.png|138| Changed|
|ComboBox.Disabled.hover.chromium.png|74| Changed|
|ComboBox.Placeholder.hover.chromium.png|361| Changed|
|ComboBox.Error.hover.chromium.png|316| Changed|
|ComboBox.Root - RTL.hover.chromium.png|374| Changed|
|ComboBox.Root - RTL.default.chromium.png|209| Changed|
|ComboBox.Root.default.chromium.png|209| Changed|
|ComboBox.Styled.default.chromium.png|201| Changed|
|ComboBox.Root.hover.chromium.png|374| Changed|
|ComboBox.Styled.hover.chromium.png|366| Changed|
|ComboBox.Disabled.click.chromium.png|74| Changed|
|ComboBox.Error.click.chromium.png|705| Changed|
|ComboBox.Root.click.chromium.png|378| Changed|
|ComboBox.Styled.click.chromium.png|372| Changed|
|ComboBox.Root - RTL.click.chromium.png|378| Changed|
|ComboBox.Placeholder.click.chromium.png|425| Changed|

</details>


  <details><summary><b>CommandBar</b> 11 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|CommandBar.Icons only.click.chromium.png|391| Changed|
|CommandBar.Icons only.hover.chromium.png|8| Changed|
|CommandBar.Root - RTL.hover.chromium.png|262| Changed|
|CommandBar.Root - RTL.default.chromium.png|281| Changed|
|CommandBar.Root - RTL.click.chromium.png|791| Changed|
|CommandBar.Root.hover.chromium.png|262| Changed|
|CommandBar.Text only.default.chromium.png|281| Changed|
|CommandBar.Text only.hover.chromium.png|258| Changed|
|CommandBar.Root.default.chromium.png|281| Changed|
|CommandBar.Root.click.chromium.png|791| Changed|
|CommandBar.Text only.click.chromium.png|687| Changed|

</details>


  <details><summary><b>ContextualMenu</b> 29 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ContextualMenu.With headers.click.chromium.png|125| Changed|
|ContextualMenu.Root.click.chromium.png|216| Changed|
|ContextualMenu.With headers.default.chromium.png|125| Changed|
|ContextualMenu.Root.hover.chromium.png|216| Changed|
|ContextualMenu.With icons.click.chromium.png|138| Changed|
|ContextualMenu.With icons.default.chromium.png|161| Changed|
|ContextualMenu.With headers.hover.chromium.png|125| Changed|
|ContextualMenu.Root.default.chromium.png|244| Changed|
|ContextualMenu.With secondaryText - RTL.click.chromium.png|200| Changed|
|ContextualMenu.With secondaryText - RTL.hover.chromium.png|200| Changed|
|ContextualMenu.With secondaryText - RTL.default.chromium.png|255| Changed|
|ContextualMenu.With secondaryText.default.chromium.png|255| Changed|
|ContextualMenu.With secondaryText.click.chromium.png|200| Changed|
|ContextualMenu.With icons.hover.chromium.png|138| Changed|
|ContextualMenu.With split button submenu.click.chromium.png|4| Changed|
|ContextualMenu.With submenu - RTL.click.chromium.png|196| Changed|
|ContextualMenu.With secondaryText.hover.chromium.png|200| Changed|
|ContextualMenu.With submenu - RTL.default.chromium.png|59| Changed|
|ContextualMenu.With submenus with hrefs.chromium.png|183| Changed|
|ContextualMenu.With split button submenu.hover.chromium.png|4| Changed|
|ContextualMenu.With split button submenu.default.chromium.png|32| Changed|
|ContextualMenu.With submenus with hrefs.item1 hovered.chromium.png|93| Changed|
|ContextualMenu.With submenu - RTL.hover.chromium.png|36| Changed|
|ContextualMenu.With submenu.click.chromium.png|196| Changed|
|ContextualMenu.With submenu.hover.chromium.png|36| Changed|
|ContextualMenu.With submenu.default.chromium.png|59| Changed|
|ContextualMenu.With submenus with hrefs.menu opened.chromium.png|36| Changed|
|ContextualMenu.With submenus with hrefs.parent hovered.chromium.png|66| Changed|
|ContextualMenu.With submenus with hrefs.item2 hovered.chromium.png|93| Changed|

</details>


  <details><summary><b>DatePicker</b> 35 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker.Allow text input.default.chromium.png|48158| Changed|
|DatePicker.Allow text input.hover datepicker.chromium.png|48338| Changed|
|DatePicker.Placeholder.default.chromium.png|48104| Changed|
|DatePicker.Required.hover datepicker.chromium.png|48282| Changed|
|DatePicker.Required.default.chromium.png|48104| Changed|
|DatePicker.Root - RTL.default.chromium.png|48104| Changed|
|DatePicker.Placeholder.hover datepicker.chromium.png|48282| Changed|
|DatePicker.Root - RTL.hover datepicker.chromium.png|48282| Changed|
|DatePicker.Root.default.chromium.png|48104| Changed|
|DatePicker.Underlined and Required.default.chromium.png|47373| Changed|
|DatePicker.Root.hover datepicker.chromium.png|48282| Changed|
|DatePicker.Underlined.hover datepicker.chromium.png|47453| Changed|
|DatePicker.Underlined.default.chromium.png|47373| Changed|
|DatePicker.Underlined and Required.hover datepicker.chromium.png|47453| Changed|
|DatePicker.Placeholder.click.chromium.png|3138| Changed|
|DatePicker.Required.click.chromium.png|3152| Changed|
|DatePicker.Root.click.chromium.png|3138| Changed|
|DatePicker.Underlined and Required.click.chromium.png|2763| Changed|
|DatePicker.Allow text input.click.chromium.png|3101| Changed|
|DatePicker.Root - RTL.click.chromium.png|3143| Changed|
|DatePicker.Underlined.click.chromium.png|2750| Changed|
|DatePicker.Allow text input.hover day.chromium.png|0| Removed|
|DatePicker.Allow text input.hover month.chromium.png|0| Removed|
|DatePicker.Placeholder.hover day.chromium.png|0| Removed|
|DatePicker.Placeholder.hover month.chromium.png|0| Removed|
|DatePicker.Required.hover day.chromium.png|0| Removed|
|DatePicker.Required.hover month.chromium.png|0| Removed|
|DatePicker.Root - RTL.hover day.chromium.png|0| Removed|
|DatePicker.Root - RTL.hover month.chromium.png|0| Removed|
|DatePicker.Root.hover day.chromium.png|0| Removed|
|DatePicker.Root.hover month.chromium.png|0| Removed|
|DatePicker.Underlined and Required.hover day.chromium.png|0| Removed|
|DatePicker.Underlined and Required.hover month.chromium.png|0| Removed|
|DatePicker.Underlined.hover day.chromium.png|0| Removed|
|DatePicker.Underlined.hover month.chromium.png|0| Removed|

</details>


  <details><summary><b>DatePicker - Disabled</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker - Disabled.With Label.hover datepicker.chromium.png|48421| Changed|
|DatePicker - Disabled.With Label.default.chromium.png|48421| Changed|
|DatePicker - Disabled.Without Label.hover datepicker.chromium.png|48357| Changed|
|DatePicker - Disabled.Without Label.default.chromium.png|48357| Changed|
|DatePicker - Disabled.Without Value.hover datepicker.chromium.png|48348| Changed|
|DatePicker - Disabled.Without Value.default.chromium.png|48348| Changed|
|DatePicker - Disabled.Without Label.click.chromium.png|616| Changed|
|DatePicker - Disabled.Without Value.click.chromium.png|710| Changed|
|DatePicker - Disabled.With Label.click.chromium.png|782| Changed|

</details>


  <details><summary><b>DatePicker - No Month Option</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.default.chromium.png|48104| Changed|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.hover datepicker.chromium.png|48282| Changed|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.click.chromium.png|2307| Changed|
|DatePicker - No Month Option.Show Month as Overlay and no Go To Today.hover day.chromium.png|0| Removed|

</details>


  <details><summary><b>DetailsHeader</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DetailsHeader.Root.default.chromium.png|53| Changed|
|DetailsHeader.Root.hoverDraggable.chromium.png|56| Changed|
|DetailsHeader.Root.hoverFrozenLast.chromium.png|53| Changed|
|DetailsHeader.Root.hoverFrozenFirst.chromium.png|53| Changed|
|DetailsHeader.Root.borderWhileDragging.chromium.png|56| Changed|
|DetailsHeader.Root.dropHint.chromium.png|56| Changed|

</details>


  <details><summary><b>DetailsList</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DetailsList.Checkbox Visible Always - RTL.hover.chromium.png|810| Changed|
|DetailsList.Checkbox Visible Always - RTL.click.chromium.png|3354| Changed|
|DetailsList.Checkbox Visible Always - RTL.default.chromium.png|777| Changed|
|DetailsList.Checkbox Visible Always.default.chromium.png|783| Changed|
|DetailsList.Checkbox Visible Always.click.chromium.png|3356| Changed|
|DetailsList.Compact.default.chromium.png|783| Changed|
|DetailsList.Compact.click.chromium.png|2731| Changed|
|DetailsList.Checkbox Visible Always.hover.chromium.png|811| Changed|
|DetailsList.Compact.hover.chromium.png|811| Changed|
|DetailsList.Grouped with Checkbox Hidden.click.chromium.png|3478| Changed|
|DetailsList.Grouped with Checkbox Hidden.default.chromium.png|929| Changed|
|DetailsList.Grouped with Checkbox Hidden.hover.chromium.png|957| Changed|
|DetailsList.Root.click.chromium.png|3356| Changed|
|DetailsList.Grouped.click.chromium.png|3655| Changed|
|DetailsList.Grouped.default.chromium.png|929| Changed|
|DetailsList.Grouped.hover.chromium.png|957| Changed|
|DetailsList.Single Selection Mode.click.chromium.png|3356| Changed|
|DetailsList.Root.default.chromium.png|783| Changed|
|DetailsList.Root.hover.chromium.png|811| Changed|
|DetailsList.Single Selection Mode.hover.chromium.png|811| Changed|
|DetailsList.Single Selection Mode.default.chromium.png|783| Changed|

</details>


  <details><summary><b>Dialog</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.Large header.default.chromium.png|47| Changed|
|Dialog.Root - RTL.default.chromium.png|330| Changed|
|Dialog.Blocking.default.chromium.png|341| Changed|
|Dialog.Root.default.chromium.png|330| Changed|
|Dialog.Wide Dialog.default.chromium.png|330| Changed|

</details>


  <details><summary><b>DocumentCard</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DocumentCard.With secondary title style.default.chromium.png|79| Changed|
|DocumentCard.Not truncated.default.chromium.png|695| Changed|
|DocumentCard.Compact with preview list.default.chromium.png|255| Changed|
|DocumentCard.Compact with preview image.default.chromium.png|462| Changed|

</details>


  <details><summary><b>Dropdown</b> 36 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown.Custom Dropdown - RTL.default.chromium.png|27650| Changed|
|Dropdown.Custom Dropdown - RTL.hover.chromium.png|27872| Changed|
|Dropdown.Custom Dropdown.default.chromium.png|27551| Changed|
|Dropdown.Custom Dropdown.hover.chromium.png|27773| Changed|
|Dropdown.Disabled option selected.default.chromium.png|21583| Changed|
|Dropdown.Disabled option selected.hover.chromium.png|21603| Changed|
|Dropdown.Multiselect - RTL.hover.chromium.png|24098| Changed|
|Dropdown.Multiselect - RTL.default.chromium.png|24099| Changed|
|Dropdown.Multiselect.hover.chromium.png|23958| Changed|
|Dropdown.Multiselect.default.chromium.png|23955| Changed|
|Dropdown.Required without label.default.chromium.png|23325| Changed|
|Dropdown.Required without label.hover.chromium.png|23496| Changed|
|Dropdown.Required.default.chromium.png|20659| Changed|
|Dropdown.Required.hover.chromium.png|20839| Changed|
|Dropdown.Custom Dropdown.hover item.chromium.png|4545| Changed|
|Dropdown.Root - RTL.default.chromium.png|25809| Changed|
|Dropdown.Root - RTL.hover.chromium.png|25975| Changed|
|Dropdown.Custom Dropdown - RTL.hover item.chromium.png|4463| Changed|
|Dropdown.Root.default.chromium.png|25741| Changed|
|Dropdown.Multiselect - RTL.hover item.chromium.png|3484| Changed|
|Dropdown.Root.hover.chromium.png|25912| Changed|
|Dropdown.Required.click.chromium.png|1608| Changed|
|Dropdown.Required without label.hover item.chromium.png|1809| Changed|
|Dropdown.Custom Dropdown - RTL.click.chromium.png|3880| Changed|
|Dropdown.Multiselect - RTL.click.chromium.png|3580| Changed|
|Dropdown.Multiselect.hover item.chromium.png|3476| Changed|
|Dropdown.Disabled option selected.hover item.chromium.png|2059| Changed|
|Dropdown.Custom Dropdown.click.chromium.png|3956| Changed|
|Dropdown.Disabled option selected.click.chromium.png|1708| Changed|
|Dropdown.Required without label.click.chromium.png|1295| Changed|
|Dropdown.Multiselect.click.chromium.png|3573| Changed|
|Dropdown.Required.hover item.chromium.png|2183| Changed|
|Dropdown.Root.click.chromium.png|2408| Changed|
|Dropdown.Root - RTL.hover item.chromium.png|2970| Changed|
|Dropdown.Root - RTL.click.chromium.png|2396| Changed|
|Dropdown.Root.hover item.chromium.png|2978| Changed|

</details>


  <details><summary><b>Dropdown Disabled</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown Disabled.Root.default.chromium.png|25471| Changed|
|Dropdown Disabled.Root.hover.chromium.png|25471| Changed|
|Dropdown Disabled.Root.click.chromium.png|621| Changed|

</details>


  <details><summary><b>Experimental Slider</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Experimental Slider.Vertical With Custom Labels.default.chromium.png|198| Changed|
|Experimental Slider.With Tickmarks.default.chromium.png|172| Changed|
|Experimental Slider.Vertical With Custom Labels.hover.chromium.png|198| Changed|
|Experimental Slider.With Tickmarks.hover.chromium.png|172| Changed|

</details>


  <details><summary><b>Facepile</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Facepile.Custom button styles.default.chromium.png|14| Changed|
|Facepile.Overflow.default.chromium.png|9| Changed|

</details>


  <details><summary><b>FocusTrapZones</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|FocusTrapZones.Dialog nested in Panel - RTL.default.chromium.png|2299| Changed|
|FocusTrapZones.Dialog nested in Panel.default.chromium.png|2299| Changed|
|FocusTrapZones.Dialog nested in Panel - RTL.click on panel close button.chromium.png|2299| Changed|
|FocusTrapZones.Dialog nested in Panel.click on panel close button.chromium.png|2299| Changed|
|FocusTrapZones.Panel on its own.default.chromium.png|320| Changed|
|FocusTrapZones.Panel on its own.click on panel close button.chromium.png|330| Changed|

</details>


  <details><summary><b>HoverCard</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|HoverCard.Root.fully expanded with test content.chromium.png|654| Changed|
|HoverCard.Root - RTL.fully expanded with test content.chromium.png|654| Changed|

</details>


  <details><summary><b>Icon</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Icon.Root.default.chromium.png|34002| Changed|

</details>


  <details><summary><b>IconButton Scenarios</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|IconButton Scenarios.icon button with menu.hover icon.chromium.png|14| Changed|
|IconButton Scenarios.normal icon button.hover icon.chromium.png|12| Changed|

</details>


  <details><summary><b>Image</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image.Fit- CenterContain, image larger.default.chromium.png|454| Changed|
|Image.Fit- CenterContain, image taller.default.chromium.png|603| Changed|
|Image.Fit- CenterContain, image smaller.default.chromium.png|638| Changed|
|Image.Fit- center, image larger.default.chromium.png|520| Changed|
|Image.Fit- CenterContain, image wider.default.chromium.png|624| Changed|
|Image.Fit- centerCover, image larger.default.chromium.png|437| Changed|
|Image.Fit- center, image smaller.default.chromium.png|805| Changed|
|Image.Fit- contain, image taller.default.chromium.png|1000| Changed|
|Image.Fit- centerCover, image taller.default.chromium.png|712| Changed|
|Image.Fit- none, image larger.default.chromium.png|727| Changed|
|Image.Fit- contain, image wider.default.chromium.png|991| Changed|
|Image.Fit- none, image smaller.default.chromium.png|869| Changed|
|Image.Fit- centerCover, image smaller.default.chromium.png|638| Changed|
|Image.Fit- centerCover, image wider.default.chromium.png|662| Changed|
|Image.Maximize frame, portrait container.default.chromium.png|18| Changed|
|Image.Maximize frame, landscape container.default.chromium.png|332| Changed|
|Image.No fit, only height.default.chromium.png|849| Changed|
|Image.Fit- cover, image taller.default.chromium.png|974| Changed|
|Image.No fit, no w-h.default.chromium.png|704| Changed|
|Image.Fit- cover, image wider.default.chromium.png|932| Changed|
|Image.No fit, only width.default.chromium.png|823| Changed|

</details>


  <details><summary><b>Label</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Label.Disabled.default.chromium.png|75| Changed|
|Label.Required.default.chromium.png|130| Changed|
|Label.Root - RTL.default.chromium.png|68| Changed|
|Label.Root.default.chromium.png|68| Changed|

</details>


  <details><summary><b>Layer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Layer.Root.default.chromium.png|70| Changed|
|Layer.Root - RTL.default.chromium.png|70| Changed|

</details>


  <details><summary><b>Link</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link.Disabled.click.chromium.png|50| Changed|
|Link.Disabled.default.chromium.png|50| Changed|
|Link.Disabled.hover.chromium.png|50| Changed|
|Link.Disabled.focus.chromium.png|50| Changed|
|Link.No Href Disabled.click.chromium.png|174| Changed|
|Link.No Href Disabled.hover.chromium.png|174| Changed|
|Link.No Href Disabled.default.chromium.png|174| Changed|
|Link.No Href Disabled.focus.chromium.png|174| Changed|
|Link.Underlined Disabled.click.chromium.png|250| Changed|
|Link.Underlined Disabled.default.chromium.png|250| Changed|
|Link.Underlined Disabled.focus.chromium.png|250| Changed|
|Link.Underlined Disabled.hover.chromium.png|250| Changed|

</details>


  <details><summary><b>MessageBar</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MessageBar.Root actions - RTL.default.chromium.png|47| Changed|
|MessageBar.Root actions single line.default.chromium.png|47| Changed|
|MessageBar.Root actions single line - RTL.default.chromium.png|47| Changed|
|MessageBar.Root dismiss and action single line - RTL.default.chromium.png|47| Changed|
|MessageBar.Root actions.default.chromium.png|47| Changed|
|MessageBar.Root dismiss and action - RTL.default.chromium.png|47| Changed|
|MessageBar.Root dismiss and action single line.default.chromium.png|47| Changed|
|MessageBar.Root dismiss and action.default.chromium.png|47| Changed|

</details>


  <details><summary><b>Modal</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Modal.Root.default.chromium.png|80| Changed|

</details>


  <details><summary><b>Nav</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Nav.Disabled.click.chromium.png|90| Changed|
|Nav.Disabled - RTL.hover.chromium.png|148| Changed|
|Nav.Disabled.hover.chromium.png|148| Changed|
|Nav.Disabled - RTL.default.chromium.png|148| Changed|
|Nav.Disabled - RTL.click.chromium.png|90| Changed|
|Nav.Disabled.default.chromium.png|148| Changed|
|Nav.Root - RTL.hover.chromium.png|329| Changed|
|Nav.Root - RTL.default.chromium.png|361| Changed|
|Nav.Root - RTL.click.chromium.png|251| Changed|
|Nav.Root.click.chromium.png|249| Changed|
|Nav.Root.default.chromium.png|359| Changed|
|Nav.Root.hover.chromium.png|326| Changed|

</details>


  <details><summary><b>OverflowSet</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|OverflowSet.Root.default.chromium.png|84| Changed|
|OverflowSet.Root - RTL.default.chromium.png|84| Changed|
|OverflowSet.Root - RTL.default.chromium_1.png|239| Changed|
|OverflowSet.Root.default.chromium_1.png|239| Changed|

</details>


  <details><summary><b>OverflowSet variant</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|OverflowSet variant.Vertical Direction.default.chromium.png|84| Changed|

</details>


  <details><summary><b>Panel</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Panel.Medium right - RTL.default.chromium.png|229| Changed|
|Panel.Large right.default.chromium.png|182| Changed|
|Panel.Large fixed right.default.chromium.png|263| Changed|
|Panel.Custom.default.chromium.png|204| Changed|
|Panel.Custom anchored left.default.chromium.png|262| Changed|
|Panel.Extra large right.default.chromium.png|259| Changed|
|Panel.SearchBox and Right Panel - RTL.click.chromium.png|1266| Changed|
|Panel.SearchBox and Right Panel - RTL.default.chromium.png|804| Changed|
|Panel.SearchBox and Right Panel.default.chromium.png|805| Changed|
|Panel.Medium right.default.chromium.png|229| Changed|
|Panel.Small fixed right w- close button - RTL.default.chromium.png|265| Changed|
|Panel.SearchBox and Right Panel.click.chromium.png|1267| Changed|
|Panel.Small left w- close button - RTL.default.chromium.png|183| Changed|
|Panel.Small left w- close button.default.chromium.png|183| Changed|
|Panel.Small fixed right w- close button.default.chromium.png|265| Changed|
|Panel.Small fluid right.default.chromium.png|261| Changed|
|Panel.With no header, close button.default.chromium.png|96| Changed|
|Panel.With no navigation.default.chromium.png|306| Changed|

</details>


  <details><summary><b>PeoplePicker</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PeoplePicker.Compact selected - RTL.default.chromium.png|16746| Changed|
|PeoplePicker.Compact selected.default.chromium.png|17242| Changed|
|PeoplePicker.List selected - RTL.default.chromium.png|11080| Changed|
|PeoplePicker.List selected.default.chromium.png|11121| Changed|
|PeoplePicker.Normal Overflow selected.default.chromium.png|17015| Changed|
|PeoplePicker.Normal Overflow selected - RTL.default.chromium.png|17068| Changed|
|PeoplePicker.Normal selected - RTL.default.chromium.png|16746| Changed|
|PeoplePicker.Normal selected.default.chromium.png|17242| Changed|
|PeoplePicker.Compact selected - RTL.suggestions.chromium.png|9658| Changed|
|PeoplePicker.Compact.suggestions.chromium.png|5834| Changed|
|PeoplePicker.List.suggestions.chromium.png|5401| Changed|
|PeoplePicker.List selected - RTL.suggestions.chromium.png|5717| Changed|
|PeoplePicker.Normal Overflow selected.suggestions.chromium.png|9771| Changed|
|PeoplePicker.Compact selected.suggestions.chromium.png|9408| Changed|
|PeoplePicker.List selected.suggestions.chromium.png|5569| Changed|
|PeoplePicker.Normal disabled with placeholder.chromium.png|29| Changed|
|PeoplePicker.Normal selected - RTL.suggestions.chromium.png|9790| Changed|
|PeoplePicker.Normal Overflow selected - RTL.suggestions.chromium.png|10002| Changed|
|PeoplePicker.Normal with text.suggestion-  a .chromium.png|5744| Changed|
|PeoplePicker.Normal selected.suggestions.chromium.png|9603| Changed|
|PeoplePicker.Normal.suggestions.chromium.png|5401| Changed|

</details>


  <details><summary><b>Persona</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Persona.Initials.default.chromium.png|11107| Changed|
|Persona.Initials - RTL.default.chromium.png|10324| Changed|
|Persona.Persona with children.default.chromium.png|11137| Changed|
|Persona.default size, details hidden.default.chromium.png|7175| Changed|
|Persona.size10 (tiny).default.chromium.png|18015| Changed|
|Persona.default size, presences.default.chromium.png|56673| Changed|
|Persona.size120.default.chromium.png|21870| Changed|
|Persona.size120 - RTL.default.chromium.png|20989| Changed|
|Persona.size32.default.chromium.png|10152| Changed|
|Persona.size24 (extraExtraSmall).default.chromium.png|17940| Changed|
|Persona.size28 (extraSmall).default.chromium.png|17097| Changed|
|Persona.size48 (regular).default.chromium.png|17468| Changed|
|Persona.size100 (extraLarge) - RTL.default.chromium.png|31987| Changed|
|Persona.size40 (small).default.chromium.png|15660| Changed|
|Persona.size72 (large).default.chromium.png|26025| Changed|
|Persona.size100 (extraLarge).default.chromium.png|33577| Changed|

</details>


  <details><summary><b>Pivot</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot.Tabs - RTL.default.chromium.png|21055| Changed|
|Pivot.Count and icon.default.chromium.png|35708| Changed|
|Pivot.Count and icon - RTL.default.chromium.png|35630| Changed|
|Pivot.Root.default.chromium.png|30882| Changed|
|Pivot.Large.default.chromium.png|30654| Changed|
|Pivot.Tabs.default.chromium.png|22046| Changed|
|Pivot.Tabs large.default.chromium.png|23856| Changed|

</details>


  <details><summary><b>ProgressIndicator</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ProgressIndicator.50% - RTL.default.chromium.png|72| Changed|
|ProgressIndicator.0%.default.chromium.png|72| Changed|
|ProgressIndicator.50%.default.chromium.png|72| Changed|
|ProgressIndicator.100%.default.chromium.png|72| Changed|

</details>


  <details><summary><b>Rating</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Rating.Large.click.chromium.png|49| Changed|
|Rating.Large.default.chromium.png|44| Changed|
|Rating.Rated.click.chromium.png|31| Changed|
|Rating.Rated - RTL.default.chromium.png|57| Changed|
|Rating.Root.click.chromium.png|31| Changed|
|Rating.Rated - RTL.click.chromium.png|31| Changed|
|Rating.Root.default.chromium.png|29| Changed|
|Rating.Rated.default.chromium.png|57| Changed|

</details>


  <details><summary><b>ResizeGroup</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ResizeGroup.Root - RTL.default.chromium.png|345| Changed|
|ResizeGroup.Root.default.chromium.png|345| Changed|
|ResizeGroup.Root.click overflow.chromium.png|513| Changed|
|ResizeGroup.Root - RTL.click overflow.chromium.png|513| Changed|

</details>


  <details><summary><b>ScrollablePane</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ScrollablePane.Default ScrollablePane Example.scrolled.chromium.png|6980| Changed|
|ScrollablePane.Default ScrollablePane Example.default.chromium.png|7233| Changed|

</details>


  <details><summary><b>ScrollablePane Details List</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ScrollablePane Details List.ScrollablePane Details List with sticky header & footer.default.chromium.png|382| Changed|
|ScrollablePane Details List.ScrollablePane Details List with sticky header & footer.scroll down to the bottom.chromium.png|382| Changed|
|ScrollablePane Details List.ScrollablePane Details List with sticky header & footer.scroll down by a small amount so that the first row is still visible.chromium.png|382| Changed|
|ScrollablePane Details List.ScrollablePane Details List with sticky header & footer.scroll up to the top.chromium.png|382| Changed|

</details>


  <details><summary><b>ScrollablePane Grouped Details List</b> 11 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.If groups are expanded, when header becomes sticky, horizontal scrollbar should be visible.chromium.png|1159| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.If groups are expanded, on horizontal scroll, over scroll should not happen for content container.chromium.png|387| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.On expanding a group, horizontal scrollbar should be visible.chromium.png|1284| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility after scrolling down to the bottom with non-zero scrollLeft.chromium.png|1510| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility when header is sticky and scrollLeft is non-zero.chromium.png|1389| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility when header is sticky.chromium.png|1350| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility after scrolling down to the bottom.chromium.png|1471| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility after scrolling up to the top with non-zero scrollLeft.chromium.png|1109| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.default- scrollbars should be visible.chromium.png|1337| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibilty after scrolling left.chromium.png|1109| Changed|
|ScrollablePane Grouped Details List.ScrollablePane scrollbars visibility.Scrollbars visibility after scrolling up to the top.chromium.png|1337| Changed|

</details>


  <details><summary><b>SearchBox</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SearchBox.Full - RTL.default.chromium.png|10912| Changed|
|SearchBox.Full - RTL.default.chromium_1.png|10688| Changed|
|SearchBox.Full.default.chromium.png|10960| Changed|
|SearchBox.Root - RTL.default.chromium.png|7698| Changed|
|SearchBox.Full.default.chromium_1.png|10681| Changed|
|SearchBox.Root.default.chromium.png|7699| Changed|
|SearchBox.Root.default.chromium_1.png|8421| Changed|
|SearchBox.Root - RTL.default.chromium_1.png|8420| Changed|

</details>


  <details><summary><b>Separator</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Separator.Horizontal End Aligned - RTL.default.chromium.png|34| Changed|
|Separator.Horizontal Start Aligned - RTL.default.chromium.png|34| Changed|
|Separator.Horizontal Start Aligned.default.chromium.png|34| Changed|
|Separator.Root - RTL.default.chromium.png|34| Changed|
|Separator.Root.default.chromium.png|34| Changed|
|Separator.Horizontal End Aligned.default.chromium.png|34| Changed|
|Separator.Vertical End Aligned - RTL.default.chromium.png|34| Changed|
|Separator.Vertical Center Aligned - RTL.default.chromium.png|34| Changed|
|Separator.Vertical End Aligned.default.chromium.png|34| Changed|
|Separator.Vertical Start Aligned - RTL.default.chromium.png|34| Changed|
|Separator.Vertical Center Aligned.default.chromium.png|34| Changed|
|Separator.Vertical Start Aligned.default.chromium.png|34| Changed|

</details>


  <details><summary><b>Signals</b> 21 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Signals.Comments (count).default.chromium.png|56| Changed|
|Signals.Blocked.default.chromium.png|48| Changed|
|Signals.Awaiting approval.default.chromium.png|101| Changed|
|Signals.Emailed.default.chromium.png|46| Changed|
|Signals.Comments.default.chromium.png|56| Changed|
|Signals.Follow.default.chromium.png|67| Changed|
|Signals.Malware detected.default.chromium.png|102| Changed|
|Signals.Missing metadata.default.chromium.png|101| Changed|
|Signals.New (positioning).default.chromium.png|10| Changed|
|Signals.Read-only.default.chromium.png|56| Changed|
|Signals.NotFollow.default.chromium.png|102| Changed|
|Signals.New.default.chromium.png|27| Changed|
|Signals.Record.default.chromium.png|40| Changed|
|Signals.Shared.default.chromium.png|40| Changed|
|Signals.Someone checked out.default.chromium.png|114| Changed|
|Signals.Trending.default.chromium.png|49| Changed|
|Signals.Unseen edit.default.chromium.png|63| Changed|
|Signals.Unseen reply.default.chromium.png|70| Changed|
|Signals.Warning.default.chromium.png|50| Changed|
|Signals.Mention.default.chromium.png|48| Changed|
|Signals.You checked out.default.chromium.png|83| Changed|

</details>


  <details><summary><b>Slider</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.EqualMinMax.default.chromium.png|111| Changed|
|Slider.Disabled.hover.chromium.png|144| Changed|
|Slider.Disabled.default.chromium.png|144| Changed|
|Slider.Max not multiple of step.hover.chromium.png|127| Changed|
|Slider.EqualMinMax.hover.chromium.png|111| Changed|
|Slider.Root - RTL.default.chromium.png|114| Changed|
|Slider.Max not multiple of step.default.chromium.png|127| Changed|
|Slider.Root - RTL.hover.chromium.png|114| Changed|
|Slider.Vertical.default.chromium.png|114| Changed|
|Slider.Step less than 1.default.chromium.png|123| Changed|
|Slider.Root.hover.chromium.png|114| Changed|
|Slider.Vertical.hover.chromium.png|114| Changed|
|Slider.Step less than 1.hover.chromium.png|123| Changed|
|Slider.Root.default.chromium.png|114| Changed|

</details>


  <details><summary><b>SpinButton</b> 28 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SpinButton.Disabled.click input.chromium.png|79| Changed|
|SpinButton.Disabled.hover arrow.chromium.png|79| Changed|
|SpinButton.Disabled.hover input.chromium.png|79| Changed|
|SpinButton.Disabled.default.chromium.png|79| Changed|
|SpinButton.Label at end with icon - RTL.default.chromium.png|163| Changed|
|SpinButton.Disabled.mouseDown arrow.chromium.png|79| Changed|
|SpinButton.Label at end - RTL.default.chromium.png|138| Changed|
|SpinButton.Label on bottom with icon.default.chromium.png|163| Changed|
|SpinButton.Label at end with icon.default.chromium.png|163| Changed|
|SpinButton.Root.default.chromium.png|138| Changed|
|SpinButton.Root.hover input.chromium.png|280| Changed|
|SpinButton.Root.click input.chromium.png|129| Changed|
|SpinButton.Root.hover arrow.chromium.png|134| Changed|
|SpinButton.With icon - RTL.hover arrow.chromium.png|159| Changed|
|SpinButton.Root.mouseDown arrow.chromium.png|284| Changed|
|SpinButton.With icon - RTL.click input.chromium.png|154| Changed|
|SpinButton.Label on top.default.chromium.png|47490| Changed|
|SpinButton.With icon.click input.chromium.png|154| Changed|
|SpinButton.With icon.default.chromium.png|163| Changed|
|SpinButton.With icon.hover arrow.chromium.png|159| Changed|
|SpinButton.With icon - RTL.hover input.chromium.png|285| Changed|
|SpinButton.With icon.mouseDown arrow.chromium.png|289| Changed|
|SpinButton.With icon.hover input.chromium.png|285| Changed|
|SpinButton.Label at end.default.chromium.png|138| Changed|
|SpinButton.With icon - RTL.default.chromium.png|163| Changed|
|SpinButton.Label on top with icon.default.chromium.png|50772| Changed|
|SpinButton.With icon - RTL.mouseDown arrow.chromium.png|289| Changed|
|SpinButton.Label on bottom.default.chromium.png|138| Changed|

</details>


  <details><summary><b>Stack</b> 44 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Stack.Horizontal Stack - Horizontally centered.default.chromium.png|27752| Changed|
|Stack.Horizontal Stack - Default - RTL.default.chromium.png|27752| Changed|
|Stack.Horizontal Stack - Gap - RTL.default.chromium.png|27752| Changed|
|Stack.Horizontal Stack - Growing items.default.chromium.png|186| Changed|
|Stack.Horizontal Stack - Default.default.chromium.png|27752| Changed|
|Stack.Horizontal Stack - Box shadow around items.default.chromium.png|26272| Changed|
|Stack.Horizontal Stack - Shrinking items.default.chromium.png|557| Changed|
|Stack.Horizontal Stack - Padding.default.chromium.png|39813| Changed|
|Stack.Horizontal Stack - Gap.default.chromium.png|27752| Changed|
|Stack.Horizontal Stack - Space around.default.chromium.png|27752| Changed|
|Stack.Horizontal Stack - Right-aligned.default.chromium.png|27752| Changed|
|Stack.Horizontal Stack - Reversed.default.chromium.png|27752| Changed|
|Stack.Horizontal Stack - Right-aligned - RTL.default.chromium.png|27752| Changed|
|Stack.Horizontal Stack - Space between.default.chromium.png|27752| Changed|
|Stack.Horizontal Stack - Wrap with specified vertical gap.default.chromium.png|6946| Changed|
|Stack.Horizontal Stack - Space evenly.default.chromium.png|27752| Changed|
|Stack.Horizontal Stack - Reversed - RTL.default.chromium.png|27752| Changed|
|Stack.Horizontal Stack - Wrap - RTL.default.chromium.png|6946| Changed|
|Stack.Horizontal Stack - Wrap.default.chromium.png|6946| Changed|
|Stack.Vertical Stack - Gap.default.chromium.png|100119| Changed|
|Stack.Vertical Stack - Default.default.chromium.png|88058| Changed|
|Stack.Vertical Stack - Default - RTL.default.chromium.png|88058| Changed|
|Stack.Horizontal Stack - Item alignments - RTL.default.chromium.png|151733| Changed|
|Stack.Horizontal Stack - Bottom-aligned.default.chromium.png|178516| Changed|
|Stack.Horizontal Stack - Item alignments.default.chromium.png|151733| Changed|
|Stack.Vertical Stack - Padding.default.chromium.png|100119| Changed|
|Stack.Vertical Stack - Item alignments - RTL.default.chromium.png|95410| Changed|
|Stack.Vertical Stack - Horizontally centered.default.chromium.png|88058| Changed|
|Stack.Vertical Stack - Item alignments.default.chromium.png|95410| Changed|
|Stack.Vertical Stack - Right-aligned - RTL.default.chromium.png|88058| Changed|
|Stack.Vertical Stack - Bottom-aligned.default.chromium.png|178516| Changed|
|Stack.Vertical Stack - Box shadow around items.default.chromium.png|206817| Changed|
|Stack.Vertical Stack - Reversed.default.chromium.png|88058| Changed|
|Stack.Vertical Stack - Reversed - RTL.default.chromium.png|88058| Changed|
|Stack.Vertical Stack - Right-aligned.default.chromium.png|88058| Changed|
|Stack.Horizontal Stack - Vertically centered.default.chromium.png|178516| Changed|
|Stack.Vertical Stack - Growing items.default.chromium.png|12061| Changed|
|Stack.Vertical Stack - Wrap - RTL.default.chromium.png|168914| Changed|
|Stack.Vertical Stack - Wrap.default.chromium.png|168914| Changed|
|Stack.Vertical Stack - Space around.default.chromium.png|178516| Changed|
|Stack.Vertical Stack - Vertically centered.default.chromium.png|178516| Changed|
|Stack.Vertical Stack - Space evenly.default.chromium.png|178516| Changed|
|Stack.Vertical Stack - Space between.default.chromium.png|178516| Changed|
|Stack.Vertical Stack - Shrinking items.default.chromium.png|24122| Changed|

</details>


  <details><summary><b>Sticky breadcrumb and sticky details list header</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.default.chromium.png|316| Changed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll down by a small amount so that the first row is still visible and the header becomes sticky.chromium.png|316| Changed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll down to the bottom.chromium.png|316| Changed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll up to the top.chromium.png|316| Changed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll back to the top when horizontal scroll is non-zero.chromium.png|316| Changed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll right.chromium.png|316| Changed|
|Sticky breadcrumb and sticky details list header.ScrollablePane Sticky Breadcrumb Details List.scroll down when horizontal scroll is non-zero.chromium.png|316| Changed|

</details>


  <details><summary><b>TagItem</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagItem.Selected - RTL.Tag Item Clear Button Hover.chromium.png|288| Changed|
|TagItem.Selected - RTL.Tag Item Select.chromium.png|205| Changed|
|TagItem.Selected - RTL.Tag Item Hover.chromium.png|211| Changed|
|TagItem.Selected - RTL.default.chromium.png|33| Changed|
|TagItem.Selected - RTL.Tag Item Clear Button Selected Hover.chromium.png|205| Changed|
|TagItem.Selected.Tag Item Clear Button Selected Hover.chromium.png|205| Changed|
|TagItem.Selected.Tag Item Hover.chromium.png|211| Changed|
|TagItem.Selected.default.chromium.png|33| Changed|
|TagItem.Selected.Tag Item Clear Button Hover.chromium.png|288| Changed|
|TagItem.Selected.Tag Item Select.chromium.png|205| Changed|

</details>


  <details><summary><b>TagPicker</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagPicker.Root.Open Suggestion Menu.chromium.png|7| Changed|
|TagPicker.Root.Suggestion Menu Item Hover.chromium.png|7| Changed|
|TagPicker.Selected.Open Suggestion Menu.chromium.png|40| Changed|
|TagPicker.Selected - RTL.Open Suggestion Menu.chromium.png|40| Changed|
|TagPicker.Selected - RTL.default.chromium.png|33| Changed|
|TagPicker.Selected.default.chromium.png|33| Changed|
|TagPicker.With long tag.default.chromium.png|96| Changed|
|TagPicker.Selected.Suggestion Menu Item Hover.chromium.png|40| Changed|
|TagPicker.Selected - RTL.Suggestion Menu Item Hover.chromium.png|40| Changed|

</details>


  <details><summary><b>TextField</b> 60 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TextField.Borderless.default.chromium.png|28433| Changed|
|TextField.Disabled.default.chromium.png|29247| Changed|
|TextField.Icon.default.chromium.png|29494| Changed|
|TextField.Icon - RTL.default.chromium.png|29494| Changed|
|TextField.Error - RTL.default.chromium.png|29272| Changed|
|TextField.Multiline - RTL.hover.chromium.png|45464| Changed|
|TextField.Multiline - RTL.click.chromium.png|45733| Changed|
|TextField.Error.default.chromium.png|29271| Changed|
|TextField.Multiline nonresizable.click.chromium.png|45733| Changed|
|TextField.Multiline - RTL.default.chromium.png|45317| Changed|
|TextField.Multiline nonresizable.default.chromium.png|45317| Changed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override - RTL.default.chromium.png|29356| Changed|
|TextField.Prefix with Value, Disabled - RTL.default.chromium.png|29295| Changed|
|TextField.Prefix with Value, Disabled.default.chromium.png|29292| Changed|
|TextField.Placeholder.default.chromium.png|29382| Changed|
|TextField.Required.default.chromium.png|29333| Changed|
|TextField.Multiline.click.chromium.png|45733| Changed|
|TextField.Multiline nonresizable.hover.chromium.png|45464| Changed|
|TextField.Placeholder - RTL.default.chromium.png|29377| Changed|
|TextField.Multiline.hover.chromium.png|45464| Changed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override.default.chromium.png|29353| Changed|
|TextField.Suffix - RTL.default.chromium.png|29289| Changed|
|TextField.Suffix.default.chromium.png|29289| Changed|
|TextField.Root.default.chromium.png|29310| Changed|
|TextField.Multiline.default.chromium.png|45317| Changed|
|TextField.Underlined.default.chromium.png|28189| Changed|
|TextField.Borderless.click.chromium.png|0| Removed|
|TextField.Borderless.hover.chromium.png|0| Removed|
|TextField.Disabled.click.chromium.png|0| Removed|
|TextField.Disabled.hover.chromium.png|0| Removed|
|TextField.Error - RTL.click.chromium.png|0| Removed|
|TextField.Error - RTL.hover.chromium.png|0| Removed|
|TextField.Error.click.chromium.png|0| Removed|
|TextField.Error.hover.chromium.png|0| Removed|
|TextField.Icon - RTL.click.chromium.png|0| Removed|
|TextField.Icon - RTL.hover.chromium.png|0| Removed|
|TextField.Icon.click.chromium.png|0| Removed|
|TextField.Icon.hover.chromium.png|0| Removed|
|TextField.Placeholder - RTL.click.chromium.png|0| Removed|
|TextField.Placeholder - RTL.hover.chromium.png|0| Removed|
|TextField.Placeholder.click.chromium.png|0| Removed|
|TextField.Placeholder.hover.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled - RTL.click.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled - RTL.hover.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override - RTL.click.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override - RTL.hover.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override.click.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled, and Prefix Style-Override.hover.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled.click.chromium.png|0| Removed|
|TextField.Prefix with Value, Disabled.hover.chromium.png|0| Removed|
|TextField.Required.click.chromium.png|0| Removed|
|TextField.Required.hover.chromium.png|0| Removed|
|TextField.Root.click.chromium.png|0| Removed|
|TextField.Root.hover.chromium.png|0| Removed|
|TextField.Suffix - RTL.click.chromium.png|0| Removed|
|TextField.Suffix - RTL.hover.chromium.png|0| Removed|
|TextField.Suffix.click.chromium.png|0| Removed|
|TextField.Suffix.hover.chromium.png|0| Removed|
|TextField.Underlined.click.chromium.png|0| Removed|
|TextField.Underlined.hover.chromium.png|0| Removed|

</details>


  <details><summary><b>Toggle</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toggle.Checked - RTL.default.chromium.png|113| Changed|
|Toggle.Unchecked - RTL.default.chromium.png|116| Changed|
|Toggle.With inline label (JSX Element).default.chromium.png|113| Changed|
|Toggle.Disabled unchecked.default.chromium.png|65| Changed|
|Toggle.Checked.default.chromium.png|113| Changed|
|Toggle.Disabled checked.default.chromium.png|62| Changed|
|Toggle.Unchecked.default.chromium.png|116| Changed|
|Toggle.With inline label and without onText and offText.default.chromium.png|96| Changed|
|Toggle.With inline label.default.chromium.png|113| Changed|
|Toggle.With inline label (JSX Element) and without onText and offText.default.chromium.png|96| Changed|
|Toggle.With inline label disabled and without onText and offText.default.chromium.png|53| Changed|
|Toggle.With inline label disabled.default.chromium.png|62| Changed|

</details>


  <details><summary><b>Tooltip</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip.Default.default.chromium.png|67| Changed|

</details>


  <details><summary><b>Tooltip - Multiple</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip - Multiple.Two Tooltips.hover inner.chromium.png|79| Changed|
|Tooltip - Multiple.Two Tooltips.hover outer.chromium.png|81| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v7"/>